README.md

```diff
@@ -1,183 +1,218 @@
+[![Build Status](https://travis-ci.org/larriereguichet/AdminBundle.svg?branch=master)](https://travis-ci.org/larriereguichet/AdminBundle)
+[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/larriereguichet/AdminBundle/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/larriereguichet/AdminBundle/?branch=master)
+[![SensioLabsInsight](https://insight.sensiolabs.com/projects/c8e28654-44c7-46f3-9450-497e37bda3d0/mini.png)](https://insight.sensiolabs.com/projects/c8e28654-44c7-46f3-9450-497e37bda3d0)
+
+
 # AdminBundle
 
-AdminBundle allows you to create flexible and robust management application, using a simple configuration,
+AdminBundle allows you to create flexible and robust backoffice application, using a simple yml configuration,
 for your Symfony application.
 
-Development and Documentation are in progress
+AdminBumble creates Admin entities (like 'product' or 'store'), to handle your Doctrine entities CRUD views. Each Admin
```

Comment:
Ease the handling of CRUD views by configuring `Admin` objects on top of your Doctrine entities. Each `Admin` has one or many `Action`. 
By default, the four actions of a classical CRUD are available (`create`, `edit`, `delete` and `list`) and the creation of new actions is painless.